README.md

```diff
@@ -43,30 +43,15 @@ The easiest way to do this is by running the interactive script
 ./scripts/domain-setup.sh
 ```
 
-#### Creating a Custom Domain
+This script:
 
-You can follow the steps at [cloud-tutorial.dev][] to get a
-custom subdomain and managed zone that are ready to use.
-
-#### Using an existing Custom Domain
-
-Your custom domain will need to be associated with a [Cloud DNS Managed Zone][]
-in the same project that you are using for this application. Be sure to
-[update name server records][] to point to your managed zone.
-
-#### Verifying Domain Ownership
-
-You must complete [domain ownership verification][] for the reference
-application to work.
-
-You can create a TXT record using the following steps:
-
-1. In [Cloud DNS][], navigate to the managed zone associated with your custom domain.
-1. Click **Add record set**.
-1. Set the **Resource Record Type** to **TXT**.
-1. In the **TXT data** field, paste the TXT record provided from following the
-   [domain ownership verification][] steps.
-1. Click **Create**.
+* Creates custom subdomains and managed zones ready for use using the steps at
```

Comment:
I think it would be helpful to explicitly point out that the script allows you to choose between creating a custom subdomain and using an existing one.

Reply:
Done